.github/workflows/retry-release.yml

```diff
@@ -0,0 +1,79 @@
+name: Retry Release
+on:
+  workflow_dispatch:
+    inputs:
+      release:
+        description: The Grails Release (e.g. 5.0.3)
+        required: true
+      target_branch:
+        description: The Target Branch (e.g. 5.0.x)
+        required: true
+permissions: {}
+jobs:
+  release:
+    permissions:
+      contents: write  #  to create release
+      issues: read  #  to get closed issues
+
+    runs-on: ubuntu-latest
+    env:
+      GIT_USER_NAME: puneetbehl
+      GIT_USER_EMAIL: behlp@unityfoundation.io
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          ref: "v${{ github.event.inputs.release }}"
+          token: ${{ secrets.GH_TOKEN }}
+      - name: Set up JDK
+        uses: actions/setup-java@387ac29b308b003ca37ba93a6cab5eb57c8f5f93 # v4
+        with:
+          distribution: 'adopt'
+          java-version: 11
+      - name: Extract Target Branch
+        id: extract_branch
+        run: |
+          echo $TARGET_BRANCH
+          echo "value=${TARGET_BRANCH}" >> $GITHUB_OUTPUT
+        env:
+          TARGET_BRANCH: ${{ github.event.inputs.target_branch }}
+      - name: Set the current release version
+        id: release_version
+        run: echo "release_version=${VERSION}" >> $GITHUB_OUTPUT
+        env:
+          VERSION: ${{ github.event.inputs.release }}
+      - name: Run Assemble
+        id: assemble
+        uses: gradle/gradle-build-action@v2
+        with:
+          arguments: assemble
+        env:
+          GRADLE_ENTERPRISE_ACCESS_KEY: ${{ secrets.GRADLE_ENTERPRISE_ACCESS_KEY }}
+          GRADLE_ENTERPRISE_BUILD_CACHE_NODE_USER: ${{ secrets.GRADLE_ENTERPRISE_BUILD_CACHE_NODE_USER }}
+          GRADLE_ENTERPRISE_BUILD_CACHE_NODE_KEY: ${{ secrets.GRADLE_ENTERPRISE_BUILD_CACHE_NODE_KEY }}
+      - name: Upload artifacts to the Github release
+        id: upload_artifact
+        if: steps.assemble.outcome == 'success'
+        uses: Roang-zero1/github-upload-release-artifacts-action@v3
+        with:
+          created_tag: v${{ github.event.inputs.release }}
+          args: build/distributions/grails-${{ steps.release_version.outputs.release_version }}.zip
```

Comment:
Please crosscheck the distribution path

Reply:
I don't think `grails-quartz` creates any files in `build/distributions/grails-*`. The artifact should be something like `build/libs/quartz-VERSION.jar`.

@Emrichardsone Were these actions created using the ones in `grails-core` as templates? Because those do a lot of things that are not needed for basic plugins.